jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l

```diff
@@ -1392,6 +1392,14 @@
       (pushback (send *baxter* arm :inverse-kinematics coords-tmp :rotation-axis t) avs)
       (send self :angle-vector-sequence avs :fast nil 0 :scale 4.0)
       ))
+  (:wait-interpolation-until-grasp
+    (arm)
+    (while (send self :interpolatingp)
+           (if (send self :graspingp arm)
+             (progn
+               (ros::ros-info "[:wait-interpolation-until-grasp] Grasping detected. Cancel angle vector: ~a" arm)
+               (send self :cancel-angle-vector))))
+    (send *baxter* :angle-vector (send self :state :potentio-vector)))
```

Comment:
I think `(send *baxter* :angle-vector (send self :state :potentio-vector)))` is not needed because `:wait-interpolation-until-grasp` is the method just for `*ri*`